src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -51,13 +48,28 @@ public ModuleClass(String className, List<Session> sessions) {
         requireAllNonNull(className, sessions);
         checkArgument(isValidModuleClassName(className), MESSAGE_CONSTRAINTS);
         this.className = className;
-        this.sessions = sessions;
+        this.sessions.setElements(sessions);
     }
 
+    /**
+     * Constructs a {@code ModuleClass} with the provided unique list of {@code Session}-s.
+     *
+     * @param className A valid class name.
+     * @param sessions A unique list of sessions.
+     */
+    public ModuleClass(String className, UniqueList<Session> sessions) {
```

Comment:
Is this constructor needed? I think current usages can be done with `List` only.